package.json

```diff
@@ -2,8 +2,7 @@
   "name": "webcrypto",
   "version": "0.1.0",
   "description": "Use the Node.js crypto module API without having to worry if it is being run in browser or Node.js",
-  "main": "lib/index.js",
```

Comment:
Really old module xD

Reply:
yeah, couldn't figure out why it was failing when linked in js-multihashing :D